bin/clover/src/pipeline-steps/assetSpecificOverrides.ts

```diff
@@ -254,6 +254,23 @@ const overrides = new Map<string, OverrideFn>([
     setAnnotationOnSocket(socket, { tokens: ["arn<string>"] });
     setAnnotationOnSocket(socket, { tokens: ["arn"] });
   }],
+  [
```

Comment:
I just realized we don't recurse into objects when making arn input sockets. This is fine for now, but we should fix that